drivers/mtd_emulated/Kconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#               2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USEMODULE_MTD_SDCARD
+    bool "Configure MTD_SDCARD driver"
+    depends on USEMODULE_MTD_SDCARD
+    help
+        Configure the MTD_SDCARD driver using Kconfig.
+
+if KCONFIG_USEMODULE_MTD_SDCARD
+
+config SDCARD_GENERIC_MTD_OFFSET
+    depends on MODULE_MTD_SDCARD_DEFAULT
+    int "Index of first auto-configured MTD SDcard device"
+    default 0
+    help
+        If you have other MTD devices defined, set this number so that
+        the auto-configured SD Card(s) from mtd_sdcard_default will come after them.
+
+config MTD_SDCARD_ERASE
+    bool "Enable SD card erase"
+    help
+        Enable this to erase sector before a data write operation.
+        SDCards handle sector erase internally so it's
+        possible to directly write to the card without erasing
+        the sector first hence this feature is disabled by default.
+
+endif # KCONFIG_USEMODULE_MTD_SDCARD
```

Comment:
MTD_SDCARD ?

Reply:
Ups, just a leftover from when I started with a copy. But it is not required at all.